ddtrace/contrib/grpc/constants.py

```diff
@@ -1,6 +1,15 @@
 import grpc
 
 
+try:
+    from grpc import aio
+
+    GRPC_AIO_PIN_MODULE_SERVER = aio.Server
+    GRPC_AIO_PIN_MODULE_CLIENT = aio.Channel
+except ImportError:
+    pass
```

Comment:
```suggestion
GRPC_AIO_PIN_MODULE_SERVER = None
GRPC_AIO_PIN_MODULE_CLIENT = None
try:
    from grpc import aio

    GRPC_AIO_PIN_MODULE_SERVER = aio.Server
    GRPC_AIO_PIN_MODULE_CLIENT = aio.Channel
except ImportError:
    pass
```

? are we going to have any issues if these are not defined?

We do use as `getattr(constants.GRPC_AIO_PIN_MODULE_SERVER, ... )`

Maybe these are better off not as constants?

e.g.

```python
def _patch_aio_client():
    if getattr(grpc.aio.Channel, "__datadog_patch", False):
        return
    setattr(grpc.aio.Channel, "__datadog_patch", True)

    Pin().onto(grpc.aio.Channel)
```

Since we only call `_patch_aio_client()` if `HAS_GRPC_AIO is True`.

what do you think?

Reply:
Updated in c716a6d. Would be great if you could have a look.
Instead of using raw `grpc.aio.Channel` and `grpc.aio.Server`, I decided to move the constants to `patch.py` as they are used in the test as well. This led to remove duplicated try-except to check if `grpc.aio` is available and make comments why they are not defined in `constants.py`.